QUICKSTART.md

```diff
@@ -0,0 +1,96 @@
+## StackHawk Setup
```

Comment:
I believe all of this information should be in HawkDocs instead, for a couple of reasons:
1. we already have hawkscan quickstart docs in hawkdocs - https://docs.stackhawk.com/hawkscan/
2. we already have hawkscan-action docs in hawkdocs - https://docs.stackhawk.com/continuous-integration/github-actions.html
3. Having the same points covered in many places will be harder to maintain than just linking to the canonical info
4. Most users of the hawkscan action won't see this repo - they will read our action's docs from the Marketplace site, which only shows the top-level README.md - https://github.com/marketplace/actions/stackhawk-hawkscan-action

I think we should link out to the hawkscan quickstart, and the hawkscan-action quick start, from the parent README.md in this repo, which gets published as the documentation in the GitHub Marketplace.